doc/03-component-interfaces/03-component-interfaces.md

```diff
@@ -132,7 +130,7 @@ hierarchy diagram using whatever tools you would like. Then, include a picture
 of it in this folder. You may also embed it just below using markdown syntax
 (i.e., `![ALT TEXT](path/to/file)`).
 
-<!-- TODO: make a diagram of your component hierarchy then delete this comment -->
+![Diagram](doc/03-component-interfaces/inventory.pdf)
```

Comment:
PDFs don't embed, nor are the committed to the repo. So, I can't see the actual file. Try a JPG or PNG.